pkgs/development/python-modules/python-fontconfig/default.nix

```
@@ -25,6 +26,16 @@ buildPythonPackage rec {
   buildInputs = [ fontconfig ];
   nativeBuildInputs = [ cython ];
 
+  patches = [
+    # distutils has been removed since python 3.12
+    # See https://github.com/vayn/python-fontconfig/pull/10
+    (fetchpatch {
+      name = "no-distutils.patch";
+      url = "https://github.com/vayn/python-fontconfig/pull/10/commits/15e1a72c90e93a665569e0ba771ae53c7c8020c8.patch";
```

```suggestion
      url = "https://github.com/vayn/python-fontconfig/commit/15e1a72c90e93a665569e0ba771ae53c7c8020c8.patch";
```

i don't think `https://github.com/vayn/python-fontconfig/pull/10/commits/15e1a72c90e93a665569e0ba771ae53c7c8020c8.patch` is stable if rebased.